pkg/ebpf/c/common/context.h

```diff
@@ -129,6 +130,23 @@ statfunc task_info_t *init_task_info(u32 tid, u32 scratch_idx)
     return bpf_map_lookup_elem(&task_info_map, &tid);
 }
 
+statfunc int init_policies_config(program_data_t *p)
+{
+    u16 version = p->config->policies_version;
+
+    // copy policies_config to event data
+    long ret = bpf_probe_read(
```

Comment:
We should start using `bpf_probe_read_kernel` and `bpf_probe_read_user` in new code that we add.
In the future we will need to fully tackle #559

Reply:
You're right, it's a bad habit.